integration_tests/__tests__/list_tests.test.js

```diff
@@ -12,10 +12,17 @@
 const runJest = require('../runJest');
 
 describe('--listTests flag', () => {
-  it('causes tests to be printed out as JSON', () => {
+  it('causes tests to be printed in different lines', () => {
     const {status, stdout} = runJest('list_tests', ['--listTests']);
 
     expect(status).toBe(0);
     expect(() => JSON.parse(stdout)).not.toThrow();
```

Comment:
this assertion does'n seem to be correct any more, since it's not a JSON object now.
we should do something like:
```js
expect(stdout.split(\n')).toHaveLength(1);
expect(stdout).toMatch('whatever.the.name.of.the.test.is.test.js');
```